2.crud_queries/tasks.md

```diff
@@ -1,38 +1,75 @@
 1. Create a database named `blog`.
 
+use blog;
+
 2. Create a collection called 'articles'.
+db.createCollection("articles");
 
 3. Insert multiple documents(at least 3) into articles. It should have fields
 
+db.articles.insertMany([{title : "one", author: "a", tag: ["express"]},{title : "two", author: "b", tag:["node"]},{title : "three", author: "c",tag : ["nodejs"]}]);
+
 4. Find all the articles using `db.COLLECTION_NAME.find()`
 
-5. Find a document using _id field.
+db.articles.find();
+
+
+5. Find a document using _id field
+
+db.articles.find({"_id" : ObjectId("5eae6cbda22992b159e9ec1d")});
 
 6. Find documents using title and author's name field.
 
+db.articles.find({title: "one", author: "a"});
+
 7. Find document using a specific tag.
 
+db.articles.find({tag :"express"});
+
 8. Update title of a document using its _id field.
 
+db.articles.update({"_id" : ObjectId("5eae6cbda22992b159e9ec1d")},{$set : {title: "ONE"}});
+
 9. Update a author's name using article's title.
 
+db.articles.update({title: "ONE"},{$set : {author : "sagar"});
+
 10. rename details field to description from articles collection. 
 
+db.articles.update({title : "ONE"},{$rename : {details : "description"}});
+
```

Comment:
This will only update the document with title ONE.
To update all documents from the collection, keep query field blank object
```
db.articles.update({},{$rename : {details : "description"}});
```